### DISCUSSION TITLE
Proper way to close a tab

### DISCUSSION THREAD

**Discussion by Haltroy** [amwx/FluentAvalonia#260] [Q&A] [answered]
Removing the tab from the list is just isn't closing it completely, It's still is in memory for some reason. What's the best and proper way to close a tab?

**Comment by amwx** [comment, marked as answer]
Created an [issue](https://github.com/amwx/FluentAvalonia/issues/261#issue-1503799509) to track this. Its a memory leak in `TabViewItem`
